examples/passport-login/src/controllers/facebook.controller.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-passport-login
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  get,
+  RestBindings,
+  Response,
+  RequestWithSession,
+} from '@loopback/rest';
+import {inject, intercept} from '@loopback/core';
+import {SecurityBindings, UserProfile} from '@loopback/security';
+
+/**
+ * Login controller for facebook
+ */
+export class FacebookController {
+  constructor(
+  ) {}
+
+  @intercept('PassportInitMW')
+  @intercept('PassportSessionMW')
+  @intercept('FacebookOauth2MW')
```

Comment:
It can be simplified as `@intercept('PassportInitMW', 'PassportSessionMW', 'FacebookOauth2MW')`.

Reply:
agreed